requirements.txt

```diff
@@ -1,3 +1,4 @@
 requests==2.28.2
 black==23.3.0
 ruff==0.0.262
+python-dotenv=1.0.0
```

Comment:
Looks like this might need to be `==`

Reply:
whoops, that's my bad. I haven't used `requirements.txt` in years. I'll fix that.